assets/src/dashboard/app/views/exploreTemplates/header/index.js

```diff
@@ -137,9 +73,7 @@ function Header({ filter, totalTemplates, search, templates, sort, view }) {
 
 Header.propTypes = {
   filter: FilterPropTypes.isRequired,
-  search: SearchPropTypes.isRequired,
   sort: SortPropTypes.isRequired,
-  templates: TemplatesPropType,
```

Comment:
templates were getting used by search, but search is never enabled here (we need an API first)